tests/tests_suite_test.go

```diff
@@ -23,6 +23,7 @@ import (
 const (
 	// TODO - maybe randomize namespace
 	testNamespace             = "ssp-operator-functests"
+	commonTemplatesTestNS     = "openshift"
```

Comment:
I think a new namespace should be created for the templates. To not pollute the existing `openshift` namespace.

Reply:
We expect users/hco to create these templates in `openshift` namespace, and this is a test, I think it's fine to pollute for this purpose

Reply:
Please note that we eventually want to execute our tier-1 suite on CNV clusters downstream, polluting a production namespace during testing is not a good idea, we need to create our own testing NS and remove it when we are done

Reply:
Added a test namespace for templates